daemon/oci_windows.go

```diff
@@ -89,9 +90,15 @@ func (daemon *Daemon) createSpec(c *container.Container) (*libcontainerd.Spec, e
 
 	// s.Windows.LayerPaths
 	var layerPaths []string
-	if img.RootFS != nil && img.RootFS.Type == "layers+base" {
+	if img.RootFS != nil && img.RootFS.Type == image.TypeLayers || img.RootFS.Type == image.TypeLayersWithBase {
```

Comment:
BTW, the parentheses I suggested change the logic. `&&` binds more strongly than `||`: https://golang.org/ref/spec#Operators